### DISCUSSION TITLE
Contributing Translations

### DISCUSSION THREAD

**Discussion by sekwah41** [sekwah41/Naruto-Mod#146] [General]
We are starting to handle translations through here https://crowdin.com/project/naruto-mod.

If you want me to add another language ready for translation feel free to message here or mention me on my discord :)

**Comment by comradekingu** [comment]
@sekwah41 I'll happily help if you use Weblate.
Despite Minecreft itself being on Crowdin, other projects have moved.

**Reply by sekwah41** [reply]
@comradekingu ill add a task on my to-do list to compare it this weekend. I know one of the other projects we have uses another tool.

**Reply by sekwah41** [reply]
Taking a look it's paid unless you self host https://weblate.org/en/hosting/ so we won't be looking to use that for now.

If we were paying for Crowdin it would probably be cheaper to switch to Weblate or Localzy

**Reply by comradekingu** [reply]
@sekwah41 There is https://hosted.weblate.org/hosting/ a bit further down.
Does this project purposefully not have a license?

**Reply by sekwah41** [reply]
Originally we were looking to add a license though we had someone stealing and claiming to have made it on tiktok which we were not sure at the time what the license would have been.

I'll be looking to switch the mod over to MIT likely now releases are being done.
